app/scripts/metamask-controller.js

```diff
@@ -2756,14 +2756,19 @@ export default class MetamaskController extends EventEmitter {
               origin,
               args.message,
             ),
-          showInAppNotification: (origin, args) =>
+          showInAppNotification: (origin, args) => {
+            const notificationArgs = { ...args };
+            notificationArgs.interfaceId = notificationArgs.content;
+            delete notificationArgs.content;
+            delete notificationArgs.type;
```

Comment:
What's going on here?

Reply:
Changing the shape to fit what the notification controller is expecting.

Reply:
Can you just create a new object, rather than spreading and deleting properties afterwards?